Functions.js

```diff
@@ -0,0 +1,22 @@
+/* В терминале оплаты сохранено ваше имя, напишите функцию для определения 
+имени в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)*/
+function getAccess(Name) {
+  return Name === "Артем" ? `Привет ${Name}` : "Нет такого имени";
+}
+console.log(getAccess("Артем"));
+
+//Напишите функцию вычисления типа аргумента и вывод типа в консоль
+const getTypeOfArgument = (a) => typeof a;
+{
+  console.log(getTypeOfArgument(3));
+}
```

Comment:
Not correct...

It can be somthing like this: `const getTypeOfArgument = arg => console.log(typeof arg);`